src/components/SelectionList/BaseSelectionList.js

```diff
@@ -169,7 +172,10 @@ function BaseSelectionList({
         }
 
         listRef.current.scrollToLocation({sectionIndex: adjustedSectionIndex, itemIndex, animated, viewOffset: variables.contentHeaderHeight});
-    };
+
+        // If we don't disable dependencies here, we would need to make sure that the `sections` prop is stable in every usage of this component.
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
This change caused a bug - [WS - Members - List of users invited to the workspace does not scroll with Up and Down arrows](https://github.com/Expensify/App/issues/29022#top)

`allOptions` is initially empty. But this callback runs only on mount, so it doesn't change when allOptions are populated.